.gitignore

```diff
@@ -30,12 +30,13 @@
 .idea/
 .vscode/
 
-# Ignore VCR cassettes
-/spec/fixtures/vcr_cassettes
+# Ignore Prison API / NOMIS VCR cassettes
+/spec/fixtures/vcr_cassettes/nomis
```

Comment:
Is `nomis` the most meaningful name for this directory?

I'm assuming lots of these VCRs also pull in data from the Community API, Keyworker API, etc.

Naming is hard, and I'm struggling to come up with a suggestion that makes sense. Maybe `local` or `untracked`? 🤔 I'm not sure.